File: LeetCode_C++/0217.存在重复元素.cpp
/*
 * @lc app=leetcode.cn id=217 lang=cpp
 *
 * [217] 存在重复元素
 */

// @lc code=start
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> st;
        for(const auto &it : nums)
        {
            if(st.count(it))
                return true;
            else
                st.insert(it);
        }
        return false;
    }
};
// @lc code=end

File: LeetCode_Java/0217.存在重复元素.java
/*
 * @lc app=leetcode.cn id=217 lang=java
 *
 * [217] 存在重复元素
 */

// @lc code=start
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> set = new HashSet<>();
        for(int num : nums) {
            if(set.contains(num)) return true;
            set.add(num);
        }
        return false;
    }
}
// @lc code=end

